Content.Server/Chemistry/ReagentEffects/WearableReaction.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Inventory;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Chemistry.ReagentEffects
+{
```

Comment:
don't use parentheses here.  look at examples of more modern systems and components to see how they are designed.

Reply:
I assume you mean that I'm using multiple parantheses? I've cleaned it up to look nicer, should be similar in look to other ReagentEffect components now.